fabric-object-builder-api-v1/src/main/java/net/fabricmc/fabric/api/object/builder/v1/client/model/FabricModelProviderHelper.java

```diff
@@ -0,0 +1,116 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.api.object.builder.v1.client.model;
+
+import net.minecraft.entity.player.PlayerEntity;
+import net.minecraft.item.CrossbowItem;
+import net.minecraft.item.ElytraItem;
+import net.minecraft.item.FishingRodItem;
+import net.minecraft.item.Item;
+import net.minecraft.item.Items;
+import net.minecraft.util.Identifier;
+
+import net.fabricmc.api.EnvType;
+import net.fabricmc.api.Environment;
+
+@Environment(EnvType.CLIENT)
+public final class FabricModelProviderHelper {
+	public static void registerCrossbow(Item crossbow) {
+		FabricModelPredicateProviderRegistry.register(crossbow, new Identifier("pull"), (itemStack4, clientWorld3, livingEntity2) -> {
```

Comment:
Not sure how I feel about this entire method. This is effectively a duplicate of all the model predicate providers in vanilla, I expect mod developers may want to do these themselves or use the rendering api possibly for rendering the item.

Reply:
Personally, and from another person, knowing to add these can be quite hidden, and for a lot of developers making basic mods with little experience, seeing something like this would 1. help them understand model providers, and 2. remove some of the difficulty in discovering them